src/pages/Lending/Pool/components/Repay/RepayConfirm.tsx

```diff
@@ -107,11 +97,13 @@ export const RepayConfirm = ({
       txId: string
       expectedCompletionTime?: number
     }) => {
+      if (!confirmedQuote) throw new Error('Cannot fetch THOR Tx status withut a confirmed quote')
+
       // Enforcing outbound checks when repaying 100% since that will trigger a collateral refund transfer
       // which we *want* to wait for before considering the repay as complete
       await waitForThorchainUpdate({
         txId: _txId,
-        skipOutbound: bn(repaymentPercentOrDefault).lt(101),
+        skipOutbound: bn(confirmedQuote.repaymentPercentOrDefault).lt(101),
```

Comment:
if we wanted to be extra paranoid we could guard against decimal amounts (e.g 100.9%)
```suggestion
        skipOutbound: bn(confirmedQuote.repaymentPercentOrDefault).lte(100),
```

Reply:
also unsure of the need to use bn for a percentage, could we use normie `number`s instead?

Reply:
This may not be clear from this diff only, but 101 is our own paranoia-enforced repayment heuristics - 100 will be transformed to 101 and anything else will stay as-is

The reason is 100% may sometimes not cut it because of the discrepancies between THOR market data and ours - especially given the fact we never refetch it in the lifecycle of a tab

Reply:
Agree regular number checks would work just as well though!